src/goto-analyzer/static_verifier.cpp

```diff
@@ -38,229 +38,224 @@ bool static_verifier(
   messaget m(message_handler);
   m.status() << "Checking assertions" << messaget::eom;
```

Comment:
Can we split this function please? It has already 250 lines.

Reply:
Ok, split